README.md

```diff
@@ -102,3 +102,11 @@ If you want to test, build and publish your changes into your own personal repo
   - GH_TOKEN, github token generated to access (tag, and push) to your github repository
   - and a set of variables that contains the docker password for each repository url ie. if you set `my.docker.repo.io,quay.io` in DOCKER_REGISTRY_LIST than you need a `my_docker_repo_io` and `quay_io` secrets with the corresponding passwords
   (Note: you should take care of GH_TOKEN and docker passwords to be non-visible secrets in Travis!)
+
+# Contributing
+
+We appreciate your help!
+
+To contribute, please read our contribution guidelines: [CONTRIBUTION.md](CONTRIBUTION.md)
+
+Note that the Static Route Operator project uses the [issue tracker](https://github.com/IBM/staticroute-operator/issues) for bug reports and proposals only. If you have questions, engage our team via Slack by [registering here](https://bxcs-slack-invite.mybluemix.net/) and join the discussion in the #general channel on our [public IBM Cloud Kubernetes Service Slack](https://ibm-container-service.slack.com/). 
```

Comment:
The slack invite link is redirecting to https://cloud.ibm.com/kubernetes/slack, maybe it is better use the newer address.

Reply:
And the https://ibm-container-service.slack.com/ is redirecting to https://ibm-cloud-success.slack.com/ :)

Reply:
done